packages/@aws-cdk/aws-ecs-patterns/README.md

```diff
@@ -158,6 +158,9 @@ const loadBalancedFargateService = new ecsPatterns.NetworkLoadBalancedFargateSer
   taskImageOptions: {
     image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
   },
+  healthCheck: {
```

Comment:
When is setting this required? I'm not sure about adding this to the default example here. If i'm looking at this documentation for the first time and i'm not super familiar with ECS, do I need to provide this? I think we should add information on:

1. What is a `healthCheck` in this context? Is it the health check from the NLB to the service?
2. When should you use this (or what use cases).
3. Maybe a link to learn more.

Reply:
I struggled with, where to put information to the README. I'm not always sure if an own section is best or just add it to an example.
Should I make a own section for the health check here? And (maybe dumb question) should it be an `h3` (`###`) heading for the section?